elasticsearch_dsl/_async/index.py

```diff
@@ -141,24 +166,26 @@ def updateByQuery(self, using=None):
             index=self._name,
         )
 
-    async def create(self, using=None, **kwargs):
+    async def create(
+        self, using: Optional[AsyncUsingType] = None, **kwargs: Any
+    ) -> None:
```

Comment:
Why `-> None`? The response could be useful I guess: https://www.elastic.co/guide/en/elasticsearch/reference/current/indices-create-index.html#create-index-wait-for-active-shards

Reply:
Yes, you are right, here I can return the response object from the low-level client.